### PR TITLE
remove some unused code of edge part

### PR DESCRIPTION
**Pull request by lvchenggang**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
Remove some unused code of edge part, which could make the project code looks neater.

Does this PR introduce a user-facing change?:
NONE

### CODE REVIEW

File: edge/pkg/devicetwin/dtmanager/twin.go [hunk 1]
```diff
@@ -23,8 +23,6 @@ const (
 	RestDealType = 0
 	//SyncDealType update form cloud sync
 	SyncDealType = 1
-	//DetailDealType detail update from cloud
-	DetailDealType = 2
```
Comment (fisherxu) [inline edge/pkg/devicetwin/dtmanager/twin.go:27]: We should not delete this one, looks like it's continuous with `SyncDealType` and `SyncTwinDeleteDealType `

**Review by fisherxu — COMMENTED** [review]
thanks for the fixing :)

File: edge/pkg/devicetwin/dtmanager/twin.go (hunk 1 above)
Comment (lvchenggang) [reply edge/pkg/devicetwin/dtmanager/twin.go]: Yes, the DetailDealType may be used later in bussiness and ut code. Has been changed back.

**Pull request merged**